### DISCUSSION TITLE
Option to import followed games

### DISCUSSION THREAD

**Discussion by Garfieldwxg55** [Mrxx99/Playnite.GameJoltLibrary#13] [Ideas]
I'm not sure if this is possible, but an option to import followed games, and perhaps even have the option to "install" them (similar to other game stores that import uninstalled games) would be fantastic!

**Comment by Mrxx99** [comment]
Importing followed games is implemented and will be released with the next version (1.2). Installing games is quite difficult, as I could either easily open the store page in the browser but downloading there is not recognized by the client or ai would have to reimplement the whole install logic from the official GameJolt client which I propably don't want to invest in and maintain

**Comment by Mrxx99** [comment]
@FivesBlue Importing games that are followed is now released with 1.2.0. (Enable it in the addon settings)

**Comment by Garfieldwxg55** [comment]
Fantastic! Thanks, just tested it out and it works great!
